.gitignore

```diff
@@ -2,9 +2,18 @@
 *~
 .*.swp
 *.pyc
+*.pyo
 *.retry
-host_vars/*
 docs/build
-hosts
 workspace
-files/*private*
+
+!__exekutir__(/.*)?/inventory/hosts
+
+jobs/.*/.*
+!jobs/.*/.empty
+
+__executir__/host_vars
```

Comment:
Should this be k (exekutir, not executir)?

I feel distinctly uncomfortable with the odd names like kommandir and exekutir and whatnot. They are too easy to misspell, and much too easy for misspellings to go uncaught: to me and a few other people, misspellings almost-literally jump out of the page. Our eye detects them automatically, effortlessly. This makes it easy to skim code looking for typos. That only works with familiar words.